sw/ip/dma/test/utils/dma_testutils.c

```diff
@@ -0,0 +1,91 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/ip/dma/test/utils/dma_testutils.h"
+
+#include "hw/top_darjeeling/sw/autogen/top_darjeeling.h"
```

Comment:
I don't think this exists yet.

Reply:
Indeed. I removed the testutils commit until the SW framework for multitop is in place.